src/app/components/bitcoin-custom-fee/bitcoin-custom-fee.tsx

```diff
@@ -138,7 +139,14 @@ export function BitcoinCustomFee({
                 />
               </Stack>
             </Stack>
-            <PreviewButton isDisabled={!props.values.feeRate} text="Use custom fee" />
+            <Button
+              data-testid={SendCryptoAssetSelectors.PreviewSendTxBtn}
+              disabled={!props.values.feeRate}
+              onClick={() => props.handleSubmit}
```

Comment:
Noticed this a bit in working on the add network fix, I believe if the button is nested in the Formik form and has type="submit" it should handle this w/out having to set the onClick here? It should use the `onSubmit` in the wrapper?

Reply:
Thanks! I tested this and indeed it seems it works without this so I removed it